README.rst

```diff
@@ -188,7 +188,7 @@ module.
 
     logging.basicConfig(level=logging.INFO)
     l = logging.getLogger('fluent.test')
-    h = handler.FluentHandler('app.follow', host='host', port=24224)
+    h = handler.FluentHandler('app.follow', host='host', port=24224, buffer_overflow_handler=handler)
```

Comment:
This example seems incorrect since handler is a module (from fluent import handler). It should be a callable.